src/material/autocomplete/autocomplete-trigger.ts

```diff
@@ -578,6 +581,12 @@ export abstract class _MatAutocompleteTriggerBase
     );
   }
 
+  /** Finalized the opening sequence once it's known that the panel will be shown. */
```

Comment:
The Finalize word throws me off a little. Can we just make this `_emitOpened`, and the description explains that emit will be done after storing some values